scss/v2/components/_promo.scss

```diff
@@ -0,0 +1,93 @@
+.promo {
+    &__container {
+        display: block;
+
+        &:focus {
+            outline: 2px solid $sun-yellow;
+            outline-offset: 2px;
+            text-decoration: none;
+        }
+
+        &:hover {
+            text-decoration: none;
+        }
+    }
+
+    &__background {
+        &--blue-gradient {
+            background-image: linear-gradient(to bottom, $night-blue, $ocean-blue);
+            color: $white;
+        }
+
+        &--plum-gradient {
+            background-image: linear-gradient(to right, $indigo-blue, $plum-purple);
+            color: $white;
+        }
+    }
+
+    &__body {
+        display: flex;
+        height: 100%;
```

Comment:
As per a later comment, I haven't run this so ignore if this is wrong - is height set to anything other than 100% elsewhere which makes this need to explicitly be declared 100%?

Reply:
From my understanding the height property being set ensures we can use the `align-items` property correctly for vertical alignment and also helps ensure better browser compatibility for older versions and IE

Reply:
Cool, happy with that then ^^